core/tests/puppeteer-acceptance-tests/blog-admin-tests/blog-admin-assign-roles-to-users-and-change-tag-properties.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
```

Comment:
The filename seems to be a bit long in my opinion. Maybe we can remove the "blog-admin" from it at least (as the information is already in the folder name).

Reply:
That sgtm, it could just be assign-roles-to-users-and-change-tag-properties.js.